docs/source/releases/v1_10_0a8.rst

```diff
@@ -0,0 +1,130 @@
+ | # # # Distribution Statement A. Approved for public release. Distribution unlimited.
+ | # # #
+ | # # # Author:
+ | # # # Naval Research Laboratory, Marine Meteorology Division
+ | # # #
+ | # # # This program is free software: you can redistribute it and/or modify it under
+ | # # # the terms of the NRLMMD License included with this program. This program is
+ | # # # distributed WITHOUT ANY WARRANTY; without even the implied warranty of
+ | # # # MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the included license
+ | # # # for more details. If you did not receive the license, for more information see:
+ | # # # https://github.com/U-S-NRL-Marine-Meteorology-Division/
+
+Please see geoips/CHANGELOG_TEMPLATE.rst for instructions on updating
```

Comment:
@jsolbrig I am going to change this myself, but just for your reference, in the "final" release notes, this blurb at the top gets deleted, we add the "Version 1.10.0a8" line (with **** underline), and a short bulleted list of the changes.